scss/bootstrap/overrides/style/_panel.scss

```diff
@@ -1,8 +1,10 @@
 .panel {
   @include box-shadow($crm-panels-box-shadow);
 
+  border-radius: 2px;
```

Comment:
in bootstrap i think there is a generic border-radius-related variable, shouldn't we use that?